frontend/src/component/executiveDashboard/ExecutiveDashboard.tsx

```diff
@@ -30,8 +43,11 @@ export const ExecutiveDashboard: VFC = () => {
                 />
             </Box>
             <StyledGrid>
-                <UserStats />
-                <FlagStats />
+                <UserStats count={executiveDashboardData.users.total || 0} />
```

Comment:
Backend is deliberately not returning null/undefined values so that we don't need to be defensive in the UI